### PR TITLE
Fix docblocks about callables

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Component/HttpKernel/Controller/ControllerResolver.php [hunk 1]
```diff
@@ -145,7 +145,7 @@ protected function doGetArguments(Request $request, $controller, array $paramete
      *
      * @param string $controller A Controller string
      *
-     * @return mixed A PHP callable
+     * @return callable A PHP callable
```
Comment (fabpot) [inline src/Symfony/Component/HttpKernel/Controller/ControllerResolver.php]: Not really here.

File: src/Symfony/Component/Process/Process.php [hunk 2]
```diff
@@ -1015,7 +1015,7 @@ private function getDescriptors()
      *
      * @param callback|null $callback The user defined PHP callback
      *
-     * @return callback A PHP callable
+     * @return \Closure A PHP closure
```
Comment (fabpot) [inline src/Symfony/Component/Process/Process.php]: Why? We do accept any valid PHP callable, no?

File: src/Symfony/Component/HttpKernel/Controller/ControllerResolver.php (hunk 1 above)
Comment (WouterJ) [inline src/Symfony/Component/HttpKernel/Controller/ControllerResolver.php]: Well, `array($instance, $method)` is returned, which is a valid PHP callable.

File: src/Symfony/Component/Process/Process.php (hunk 2 above)
Comment (nicolas-grekas) [inline src/Symfony/Component/Process/Process.php]: it's the return value, always a closure yes

**Comment by xabbuh** [comment]
:+1: 

Status: Reviewed

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**